src/main/java/com/powsybl/openloadflow/sensi/AcSensitivityAnalysis.java

```diff
@@ -240,15 +240,6 @@ public void analyse(Network network, List<PropagatedContingency> contingencies,
                 slackParticipationByBus = Collections.singletonMap(lfNetwork.getSlackBus(), -1d);
             }
 
-            // if we have at least one bus target voltage linked to a ratio tap changer, we have to rebuild the AC equation
```

Comment:
@geofjamg I have removed these lines because we do it in the cleanup, but it is another issue, that we have to handle after this one... do I keep this removal?